packages/ui/src/preset-item.tsx

```diff
@@ -79,41 +45,45 @@ export class PresetItem {
     });
   }
 
-  get playerConfig(): KalturaPlayerPresetComponent | null {
-    const containerName = getPlayerPresetContainer(
-      this._options.data.container
-    );
-    const {relativeTo} = this._options.data;
+  get playerConfig(): KalturaPlayerPresetComponent[] {
+    const configs: KalturaPlayerPresetComponent[] = [];
 
-    if (!containerName) {
-      this._logger.warn(`unknown container requested`, {
-        method: 'playerConfig',
-      });
-      return null;
-    }
+    for (const presetType in this._options.data.presetAreas) {
```

Comment:
presetType > presetName?

Reply:
the current naming convention is:
presetName: Live, Playback...
presetType: playerArea, presetArea...
container: the actual preset element container name (currently similar to presetType)